content/blog/orderly-1.3.0.md

```diff
@@ -0,0 +1,29 @@
+---
+author: "Rich FitzJohn"
+title: "Orderly 1.3.0 released to CRAN"
+date: 2021-06-21
+draft: true
+tags:
+ - orderly
+ - reproducible research
+---
+
+After 18 months of heavy use within [VIMC](https://www.vaccineimpact.org/) and in the [COVID-19 modelling response](https://www.imperial.ac.uk/mrc-global-infectious-disease-analysis/covid-19/covid-19-response-team-2020-2021-report/), we have released orderly 1.3.0 to CRAN. This is a major update with many new features (more than 100 PRs since version 1.0.4 was published in early 2020).
+
+Orderly is our [reproducible research framework](https://www.vaccineimpact.org/orderly/articles/orderly.html), designed to keep track of multiple versions of reports and allow researchers to automatically audit and roll back their data, or construct workflows around long running tasks.
+
+Since 2020 we've added:
+
+* Expanded documentation, including [a discussion of different workflow patterns orderly enables](https://www.vaccineimpact.org/orderly/articles/patterns.html)
+* Support for exporting work to be rerun elsewhere (e.g., a HPC) and then sent back to a central server via [bundles](https://www.vaccineimpact.org/orderly/articles/bundles.html)
+* Massively improved the power of [OrderlyWeb](https://github.com/vimc/orderly-web/), which can be used as a central server to share artefacts among a team [see the remotes vignette](https://www.vaccineimpact.org/orderly/articles/remote.html).
+* Expanded how dependencies work, to allow depending on the latest version of a particular parameter (e.g., `latest(parameter:n_samples >= 1000)`) rather than just the latest version
+* Add new report development mode [`orderly_develop_start`, `orderly_develop_status` and `orderly_develop_clean`](https://www.vaccineimpact.org/orderly/reference/orderly_develop_start.html) which will setup environment so reports can be developed in the same was as one might write code outside of orderly. It copies required files, source code files, load declared packages. This supersedes `orderly_test_start` but function left in for now.
```

Comment:
*Added* new report...
Perhaps setup *an* environment
load -> *loads* declared packages
perhaps [the] function [is] left in for now